apps/server/test-i9n/defichain/wallet.i9n.ts

```diff
@@ -13,7 +13,7 @@ describe('DeFiChain Wallet Integration Testing', () => {
   beforeAll(async () => {
     startedHardhatContainer = await new HardhatNetworkContainer().start();
     hardhatNetwork = await startedHardhatContainer.ready();
-    testing = new BridgeServerTestingApp(TestingExampleModule.register(startedHardhatContainer));
+    testing = new BridgeServerTestingApp(TestingExampleModule.register(buildTestConfig(startedHardhatContainer, '')));
```

Comment:
```suggestion
    testing = new BridgeServerTestingApp(TestingExampleModule.register(buildTestConfig(startedHardhatContainer)));
```

Since it's optional, we don't need to pass it in. Same for the other test.